test/pytest/service-desktop/test_user_data_backup_restore.py

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
```

Comment:
it's a valid code, could you please move it to repository https://github.com/mudita/QAMuditaOS and ask @filkamola for adoption there? otherwise, our QA won't run it, and after moving - there is a chance :)